scripts/postprocess-docs.py

```diff
@@ -0,0 +1,41 @@
+import json, os, re, fileinput, getopt, sys, urllib.request
+
+base_url=''
+docs_root=''
+
+try:
+  opts, args = getopt.getopt(sys.argv[1:], "b:d:")
+except getopt.GetoptError:
+  print('post-document.py -b <base_url> -d <docs_root>')
+  sys.exit(2)
+for opt, arg in opts:
+  if opt in ("-b", "--baseurl"):
+     base_url = arg
+  elif opt in ("-d", "--docsroot"):
+     docs_root = arg
+     
+# receive, parse and remove index file
+with urllib.request.urlopen('{base_url}/search.json'.format(base_url=base_url)) as url:
+  index_data = json.loads(url.read().decode())
```

Comment:
Oh, maybe we can go this way?
```suggestion
  index_data = json.load(url)
```